docs/book/toc.md

```diff
@@ -24,6 +24,7 @@
 * [Integrations](advanced-guide/integrations.md)
 * [Track Metadata in the Cloud](advanced-guide/track_metadata_in_the_cloud.md)
 * [Execute Pipelines in the Cloud](advanced-guide/execute-pipelines-in-cloud.md)
+	* [Getting Started on GCP](advanced-guide/gcp_entrypoint.md)
```

Comment:
Maybe best if we keep the markdown file name in line with the heading, otherwise it'll be part of @bcdurak's next refactoring cycle.